admin-dev/themes/new-theme/js/components/showcase-card/extension/showcase-card-close-extension.ts

```diff
@@ -37,7 +37,7 @@ export default class ShowcaseCardCloseExtension {
    */
   extend(helperBlock: ShowcaseCard): void {
     const container = helperBlock.getContainer();
-    container.on('click', '.js-remove-helper-block', (evt) => {
+    container.on('click', '.js-remove-helper-block', (evt: { target: any; }) => {
```

Comment:
```suggestion
    container.on('click', '.js-remove-helper-block', (evt: Event) => {
```
Maybe not the right type, but isn't there a native type for events that could be used here?

Reply:
ok done, replace with `JQuery.Event`